cob_base_velocity_smoother/package.xml

```diff
@@ -13,12 +13,8 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
-  <build_depend>geometry_msgs</build_depend>
-  <build_depend>roscpp</build_depend>
-  <build_depend>std_msgs</build_depend>
-
-  <run_depend>geometry_msgs</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>std_msgs</run_depend>
+  <depend>geometry_msgs</depend>
+  <depend>roscpp</depend>
+  <depend>std_msgs</depend>
```

Comment:
this package explicitely [uses boost](https://github.com/ipa320/cob_control/blob/indigo_dev/cob_base_velocity_smoother/include/cob_base_velocity_smoother/cob_base_velocity_smoother.h#L62-L63)

Should we have this as an explicit system dependency or leave it as it is implicitly imported through `roscpp` anyways?